### ISSUE TITLE
Custom Fov?

### ISSUE THREAD

**Issue by KieranCarden** [7leodev/TimeOfDay#15]
When using the Sky mode, the `Sky Custom Fov` can be utilized to change the horizon line position. This value unfortunately isn't taken into account when using Color+Sky.

Is it possible to add a Fov slider in the Global properties of Skydome to change the horizon? Or to take into account the value from from the world environment node?

**Comment by 7leodev** [comment]
You can change the horizon altitude in: Atmosphere->LevelParam-> Z value. But not yet for the clouds.

**Comment by KieranCarden** [comment]
Whoops, meant to respond to this. Thanks for that information, exactly what I was looking for.

With my current camera angle and general style I'm going for, the clouds aren't as important (for me), but out of curiosity are the clouds something you're planning for a future release?

**Comment by Jay7c** [comment]
@KieranCarden Yes, for godot 4 I plan to add volumetric clouds. The design of the plugin will also change completely.
